### PR TITLE
Improve code snippets

### PR DESCRIPTION
**Pull request by tbedford**
## Description

Improve code snippets:

- [x] Fix error handling to something that works
- [x] Add support for sandbox for FB, Viber, WA
- [x] Add gitignore for local testing
- [x] Test via sandbox where sandbox supports it (done for FB, Viber, WA)

## Review

[Review app](https://nexmo-developer-pr-2671.herokuapp.com/messages/code-snippets/viber/send-text)

### CODE REVIEW

File: messages/send-mms.js [hunk 1]
```diff
@@ -18,13 +19,19 @@ const nexmo = new Nexmo({
...
     "content": {
       "type": "image",
-       "image": { "url": "IMG_URL" }
+       "image": { "url": IMAGE_URL }
     }
   },
-  (err, data) => { console.log(data.message_uuid); }
+  (err, data) => {
+    if (err) {
+      console.log("logging after error !!!", err);
```
Comment (AlexLakatos) [inline messages/send-mms.js]: Most code snippets use `console.error(err)` here.

**Review by AlexLakatos — APPROVED** [review]
All looks good except for the small cosmetic change for consistency on the error logging, but I can live with it as is.

File: messages/send-mms.js (hunk 1 above)
Comment (tbedford) [reply messages/send-mms.js]: While I'm in there I should improve consistency. I've committed an update.

Comment (tbedford) [reply messages/send-mms.js]: Hopefully that fixes the issue?

**Review by AlexLakatos — APPROVED** [review]
It does indeed!

**Pull request merged**